### DISCUSSION TITLE
Let's talk about great ideas

### DISCUSSION THREAD

**Discussion by formidablae** [formidablae/formidablae#34] [Ideas]
## 👋 Hi there!
To get started, comment below with some great idea you would like to discuss, maybe new features, projects or just shower thoughts :smile:.

**Comment by shegz101** [comment]
Hello, I am segun by name. The idea I would love to discuss about, is an app that works just like Spotify and sound cloud but an added feature of an embedded platform that connects upcoming artiste to music producers, beat makers. It is a music website that would be use for streaming music's, the major aim is to have a platform that can showcase upcoming artiste to the world, upcoming artiste that have little or no money to promote themselves (by getting an influencer, or promoter)

**Reply by formidablae** [reply]
Pleased to know you Segun :)

Hmmm that might really be helpful to wannabe artists. So if I understand it correctly, you intend to directly connect upcoming artists and producer studios in the same way LinkedIn connects professionals to company recruiters? But instead of resumes, upcoming artists would have to show how talented they are with samples of their work/songs, right?

**Reply by shegz101** [reply]
Yes, you really understand the idea. And I like the analogy of LinkedIn (that explains my idea better).
